src/renderer/services/file-management-system/index.ts

```diff
@@ -547,13 +596,14 @@ export default class FileManagementSystem {
       chunkNumber += 1;
       uploadChunkPromises.push(uploadChunk(chunk, chunkNumber, md5ThusFar));
     }
-
+    const offset = chunkSize * initialChunkNumber;
+    //TODO SWE-865 only read if offset < fssStatus
```

Comment:
Would this codepath be triggered under this condition?

Reply:
The behavior of the tests suggests it would.  I tried asserting that `fileReader.read` is not called and that failed.